### PR TITLE
adjust image_flip launch and config files

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

**Comment by NadiaHG** [comment]
please, merge together with https://github.com/ipa320/cob_perception_common/pull/47

File: cob_bringup/drivers/image_flip.launch [hunk 1]
```diff
@@ -2,16 +2,15 @@
...
+
   <!-- sensor message gateway node (forwards sensor messages in a desired rate) -->
-  <node name="kinect_image_flip_$(arg name)" pkg="cob_image_flip" ns="/cob_image_flip" type="kinect_image_flip" output="screen"> <!--launch-prefix="valgrind"-->
-    <rosparam command="load" file="$(find cob_hardware_config)/$(arg robot)/config/image_flip_params_$(arg name).yaml"/>
-    <param name="robot" type="string" value="$(arg robot)"/>
-    <remap from="~pointcloud_in" to="/$(arg name)/rgb/points"/>
-    <remap from="~pointcloud_out" to="/$(arg name)/flip/rgb/points"/>
-    <remap from="~colorimage_in" to="/$(arg name)/rgb/image_raw"/>
-    <remap from="~colorimage_out" to="/$(arg name)/flip/rgb/image_raw"/>
+  <node name="image_flip_$(arg camera_name)" pkg="cob_image_flip" ns="/cob_image_flip" type="image_flip" output="screen">
```
Comment (fmessmer) [inline cob_bringup/drivers/image_flip.launch:8]: can we use ns, pkg, type, name, others... as default order in the node tag?

**Comment by fmessmer** [comment]
Can we use a consistent order of launch file arguments?
e.g.
1. robot
2. camera_name
3. nodelet_manager
4. start_manager

...and then use it in all launch files (image_flip.launch, image_flip_nodelet.launch) and also where those launch files are called, i.e. the _robot_.xml/launch

**Comment by fmessmer** [comment]
I cherry-picked this commit into #279, resolved merge conflicts and then harmonized the files according to my comments here....

Thus, #279 should automatically close this one, too...

**Comment by floweisshardt** [comment]
done in #282

**Pull request closed without merging**